py3status/modules/do_not_disturb.py

```diff
@@ -90,13 +90,14 @@ class Dunst(Notification):
     """
 
     def toggle(self, state):
-        # if not in pause mode we delete all pending notifications in the queue
-        # before resuming
-        if self.parent.pause is False and state is False:
-            self.parent.py3.command_run("killall --signal SIGTERM dunst")
-        self.parent.py3.command_run(
-            "killall --signal {} dunst".format(self.parent.signals[state])
-        )
+        if state:
+            self.parent.py3.command_run("pkill -SIGUSR1 dunst")  # pause
+        else:
+            if not self.parent.pause:
+                # if not in pause mode we delete all pending notifications in
+                # the queue before resuming
+                self.parent.py3.command_run("pkill -SIGTERM dunst")
+            self.parent.py3.command_run("pkill -SIGUSR2 dunst")  # resume
```

Comment:
@ultrabug I know you wanted an approach based on USR signals, but I've just tested and in practice `-SIGTERM` doesn't just delete pending notifications, but it _also_ resumes notifications. So the comment is a bit misleading, and `-SIGUSR2` is useless waste of cycles when `-SIGTERM` was already issued. 

How do you feel about this snippet?

```python
if self.parent.pause:
    self.parent.py3.command_run("pkill -x -SIGUSR2 dunst")  # resume
else:
    self.parent.py3.command_run("pkill -x -SIGTERM dunst")  # delete all pending notifications and resume
```

Reply:
@maximbaz I feel okay with this, thanks for testing and pointing this out. **BUT** that's for another PR of course! ;)